apps/package.json

```diff
@@ -94,7 +95,7 @@
     "http-proxy": "1.11.1",
     "http-server": "^0.9.0",
     "immutable": "3.8.1",
-    "johnny-five": "bcjordan/johnny-five#with-firmata-fix",
+    "johnny-five": "^0.9.62",
```

Comment:
For both of these, should we pin to specific versions rather than using the caret (and needing to worry about different devs ending up on slightly different versions)?